js/views/pauseScreen.js

```diff
@@ -20,6 +20,9 @@ var PauseScreen = React.createClass({
   mixins: [ResizeMixin, AnimateMixin],
 
   getInitialState: function() {
+    this.handlePlayerMouseMove = this.handlePlayerMouseMove.bind(this);
```

Comment:
`React.createClass` will auto-bind, I think this is not necessary

Reply:
\+